src/RuntimeGeneratedFunctions.jl

```diff
@@ -44,9 +51,13 @@ end
 """
 struct RuntimeGeneratedFunction{argnames, cache_tag, context_tag, id} <: Function
     body::Expr
-    function RuntimeGeneratedFunction(cache_tag, context_tag, ex)
+    function RuntimeGeneratedFunction(cache_tag, context_tag, ex; opaque_closures=true)
```

Comment:
doesn't this need to be set based on the Julia version?

Reply:
See the check just below. It's ignored if `Base.Experimental.@opaque` is not defined.